source/intro.rst

```diff
@@ -127,10 +127,10 @@ Requirements
 
 Command line examples were written on an `Ubuntu`_ 12.10 desktop environment.
 Everything we do in this book can also be done on other flavors of Linux, on Mac
-OSX - probably even on Windows.
+OSX—probably even on Windows.
```

Comment:
Not a fan of the em-dash in plain text files.  Also not a fan of removing the spaces - it makes the sentence harder to read.  

Please remove this change.